lib/taskana-core/src/test/java/acceptance/task/claim/ClaimTaskAccTest.java

```diff
@@ -229,149 +223,4 @@ void should_ForceCancelClaimTask_When_InReviewByAnotherUser() throws Exception {
     assertThat(unclaimedTask.isRead()).isTrue();
     assertThat(unclaimedTask.getOwner()).isNull();
   }
-
-  @WithAccessId(user = "user-1-2")
-  @Test
-  void should_CompleteTask_When_TaskIsClaimed() throws Exception {
-    final Instant before = Instant.now().minus(Duration.ofSeconds(3L));
-    TaskService taskService = taskanaEngine.getTaskService();
-    Task claimedTask = taskService.getTask("TKI:000000000000000000000000000000000032");
-
-    taskService.completeTask(claimedTask.getId());
-
-    Task completedTask = taskService.getTask("TKI:000000000000000000000000000000000032");
-    assertThat(completedTask).isNotNull();
-    assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
-    assertThat(completedTask.getCompleted()).isNotNull();
-    assertThat(completedTask.getModified()).isEqualTo(completedTask.getCompleted());
-    assertThat(completedTask.getCompleted().isAfter(before)).isTrue();
-    assertThat(completedTask.getModified().isAfter(before)).isTrue();
-    assertThat(completedTask.isRead()).isTrue();
-    assertThat(completedTask.getOwner()).isEqualTo("user-1-2");
-  }
-
-  @WithAccessId(user = "user-1-2")
-  @Test
-  void should_CompleteTask_When_TaskIsInReview() throws Exception {
-    final Instant before = Instant.now().minus(Duration.ofSeconds(3L));
-    TaskService taskService = taskanaEngine.getTaskService();
-    Task claimedTask = taskService.getTask("TKI:400000000000000000000000000000000028");
-
-    taskService.completeTask(claimedTask.getId());
-
-    Task completedTask = taskService.getTask("TKI:400000000000000000000000000000000028");
-    assertThat(completedTask).isNotNull();
-    assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
-    assertThat(completedTask.getCompleted()).isNotNull();
-    assertThat(completedTask.getModified()).isEqualTo(completedTask.getCompleted());
-    assertThat(completedTask.getCompleted().isAfter(before)).isTrue();
-    assertThat(completedTask.getModified().isAfter(before)).isTrue();
-    assertThat(completedTask.isRead()).isTrue();
```

Comment:
This assertion should be mainly relevant for "claim" tests. After claiming, a Task has automatically been read. That's probably why it's in this test. I will still add it to complete tests because forceComplete can be done from the READY state, so that the Task hasn't been read yet.